apps/pie-microsite/src/assets/styles/base/_layout.scss

```diff
@@ -6,14 +6,15 @@
     grid-template: 64px 1fr max-content / 260px 1fr;
     grid-template-areas:
         'header header'
-        'nav main'
-        'nav footer';
+        'main main'
```

Comment:
Is this the same?
```
grid-template-areas:
    'header'
    'main'
    'footer'
```

Reply:
Not quite. It currently still uses to a 2 column grid on mobile and desktop (your code is a single column grid). We can change this in future if we need to but I'd prefer to leave it for this PR if that's ok

Reply:
Ah right, I thought this was just being used for the layout for the top level components (header, main, footer), and we would specify the column layout inside those using `grid-template-columns`, for example.